src/snowflake/cli/_plugins/nativeapp/manager.py

```diff
@@ -811,8 +813,20 @@ def get_events(
         if not self.account_event_table:
             raise NoEventTableForAccount()
 
-        # resource_attributes:"snow.database.name" uses the unquoted/uppercase app name
+        # resource_attributes uses the unquoted/uppercase app and package name
         app_name = unquote_identifier(self.app_name)
+        package_name = unquote_identifier(self.package_name)
```

Comment:
why are we unquoting these? What if there are unsafe SQL characters (like `.` for example)

Reply:
these aren't being used as identifiers, they're being compared as string literals, e.g. 
```sql
resource_attributes:"snow.application.package.name" = 'MY_PACKAGE.WITH_DOTS'
```